executor/insert_common.go

```diff
@@ -518,8 +518,8 @@ func (e *InsertValues) getRow(ctx context.Context, vals []types.Datum) ([]types.
 	hasValue := make([]bool, len(e.Table.Cols()))
 	for i := 0; i < e.rowLen; i++ {
 		casted, err := table.CastValue(e.ctx, vals[i], e.insertColumns[i].ToInfo(), false, false)
-		if e.handleErr(nil, &vals[i], 0, err) != nil {
-			return nil, err
+		if er := e.handleErr(nil, &vals[i], 0, err); er != nil {
+			return nil, er
```

Comment:
```suggestion
		if err := e.handleErr(nil, &vals[i], 0, err); err != nil {
			return nil, err
```

Is this PR ready for review?

Reply:
@tisonkun, Thanks for your attention, this problem is quite minor and bearable currently. so in the near future, i will pay no attention on it and it's still in draft.